docs/UserGuide.md

```diff
@@ -216,15 +216,15 @@ Format: `help`
 
 ##### Clearing all entries : `clear`
```

Comment:
Should we specify clear person or clear appt?

Reply:
Sure, I'll do that